Dockerfile

```diff
@@ -1,32 +1,65 @@
-FROM node:0.12
+# Build:
+# docker build -t lirantal/meanjs .
```

Comment:
My bad, you're right.
I initially created a user repository on docker hub for myself but since then I already created meanjs organization repository: https://hub.docker.com/r/meanjs/mean/

I anyway want to enhance some other things so I'll do it in another PR